src/main/kotlin/uk/gov/justice/digital/hmpps/dpssmoketest/config/WebClientConfiguration.kt

```diff
@@ -58,6 +59,25 @@ class WebClientConfiguration(
       .build()
   }
 
+  @Bean
+  fun probationOffenderSearchWebClient(authorizedClientManager: OAuth2AuthorizedClientManager): WebClient {
+    val oauth2Client = ServletOAuth2AuthorizedClientExchangeFilterFunction(authorizedClientManager)
+    oauth2Client.setDefaultClientRegistrationId("probation-offender-search")
+
+    return webClientBuilder
+      .baseUrl(probationOffenderSearchUri)
+      .apply(oauth2Client.oauth2Configuration())
+      .exchangeStrategies(
+        ExchangeStrategies.builder()
+          .codecs { configurer ->
+            configurer.defaultCodecs()
+              .maxInMemorySize(-1)
```

Comment:
I didn't realise we were still doing this for the webClients - I think we can drop the exchange strategy and just add the property `spring.codec.max-in-memory-size: 10MB`